clients/fides-js/src/lib/tcf/constants.ts

```diff
@@ -1,5 +1,7 @@
 import { TcfExperienceRecords, TcfModelType } from "./types";
 
+export const FIDES_SEPARATOR = ",";
```

Comment:
This probably need a comment, just to explain why the "Fides TC String" may include both the IAB TCF string and the AC string, together. The TL;DR of that is because we want a single-source-of-truth for offline storage & syncing of these kinds of preference strings, but we can't wait for GPP to solve that so we can approximate it here.

Important to also state that this doesn't affect the standards-compliant JS APIs that fetch TCF and AC strings separately!